### DISCUSSION TITLE
Success for LMU540HV and LMCN077HV and LMCN185HV Ceiling Cassettes

### DISCUSSION THREAD

**Discussion by kkrankall** [JanM321/esphome-lg-controller#64] [Show and tell]
I have a LG LMU540HV Multi-F Max ODU and 3x LMCN077HV and 1x LMCN185HV Ceiling Cassettes. Everything is working great. I'm not using anything for my two wall mounted units as they are ThinQ enabled and the company that I commissioned to turn my system on fried the remote ports by applying mains voltage to them.

I get a lot of these warning but seams to work fine.
[13:05:12][W][component:237]: Component lg_controller.climate took a long time for an operation (516 ms).
[13:05:12][W][component:238]: Components should block for at most 30 ms.

What would also be great if we could create presets like home, sleep, away etc. then this way here can just set up a simple schedule in HA.

Thank you for this great write up.

**Comment by JanM321** [comment]
Thanks for the report. You could add these units to the wiki or I can do it when I have some time.

> I get a lot of these warning but seams to work fine.

That's expected because we monitor if the line is idle for a while before trying to send a message. Maybe this could be implemented differently but it's likely more complicated and this works well in practice.

> What would also be great if we could create presets like home, sleep, away etc. then this way here can just set up a simple schedule in HA.

The best way to do that is in Home Assistant. Eg I have some automations to change the setpoint or to turn off the AC based on some conditions.

**Reply by kkrankall** [reply]
Jan thank you again for all of your work. It's working great. I would like to get your option. In my living room I have a premta200 and as you know you can't put that remote into slave mode. My problem is both the indoor units thermistor and the premta200's thermistor are way off and widely inconsistent. 

Do you know a way to connect/hijack to the indoor units thermistor and feed it temperatures from another sensor?

Thank you in advance.

**Reply by JanM321** [reply]
> Do you know a way to connect/hijack to the indoor units thermistor and feed it temperatures from another sensor?

You could check if your unit supports connecting an external thermistor to the PCB. Or move/extend the thermistor somehow but I've never done that and it seems risky.

Another option is to replace the premta200 with something based on Home Assistant.

**Reply by kkrankall** [reply]
I bought an external thermistor a few years ago. Now I just have to find it. 

Thank you again
